float32 diff_angle
float64 position_x
float64 position_y
float64 position_z
float64 orientation_w
float64 orientation_x
float64 orientation_y
float64 orientation_z

